recipe/meta.yaml

```diff
@@ -24,19 +21,35 @@ requirements:
 
 test:
   commands:
+    # binaries
     - mdb_copy -V                                    # [unix]
     - mdb_dump -V                                    # [unix]
     - mdb_load -V                                    # [unix]
     - mdb_stat -V                                    # [unix]
-    - test -f ${PREFIX}/lib/liblmdb.a                # [unix]
+
+    # shared:
     - test -f ${PREFIX}/lib/liblmdb.so               # [unix]
+    - if not exist %LIBRARY_BIN%\lmdb.dll exit 1     # [win]
+
+    # TODO: do we want to do have static libraries at all?
+    # static:
+    - test -f ${PREFIX}/lib/liblmdb.a                # [not win]
```

Comment:
No we should remove this per [CFEP 18]( https://github.com/conda-forge/cfep/blob/2409812d68296fcf2e06255687cc7cb085923ffd/cfep-18.md )

Reply:
OK.